FreeBSD regression

Hi!

Currently this crate doesn't build on FreeBSD:
```
   Compiling backtrace-sys v0.1.21 (file://[redacted]/backtrace-rs/backtrace-sys)
error: failed to run custom build command for `backtrace-sys v0.1.21 (file://[redacted]/backtrace-rs/backtrace-sys)`
process didn't exit successfully: `[redacted]/backtrace-rs/target/debug/build/backtrace-sys-baa1743a33b2fd1e/build-script-build` (exit code: 101)
--- stdout
TARGET = Some("x86_64-unknown-freebsd")
OPT_LEVEL = Some("0")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
CC_x86_64-unknown-freebsd = None
CC_x86_64_unknown_freebsd = None
HOST_CC = None
CC = None
HOST = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
CFLAGS_x86_64-unknown-freebsd = None
CFLAGS_x86_64_unknown_freebsd = None
HOST_CFLAGS = None
CFLAGS = None
DEBUG = Some("true")
running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-I" "src/libbacktrace" "-I" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out" "-fvisibility=hidden" "-DBACKTRACE_ELF_SIZE=64" "-DBACKTRACE_SUPPORTED=1" "-DBACKTRACE_USES_MALLOC=1" "-DBACKTRACE_SUPPORTS_THREADS=0" "-DBACKTRACE_SUPPORTS_DATA=0" "-DHAVE_DL_ITERATE_PHDR=1" "-D_GNU_SOURCE=1" "-D_LARGE_FILES=1" "-Dbacktrace_full=__rbt_backtrace_full" "-Dbacktrace_dwarf_add=__rbt_backtrace_dwarf_add" "-Dbacktrace_initialize=__rbt_backtrace_initialize" "-Dbacktrace_pcinfo=__rbt_backtrace_pcinfo" "-Dbacktrace_syminfo=__rbt_backtrace_syminfo" "-Dbacktrace_get_view=__rbt_backtrace_get_view" "-Dbacktrace_release_view=__rbt_backtrace_release_view" "-Dbacktrace_alloc=__rbt_backtrace_alloc" "-Dbacktrace_free=__rbt_backtrace_free" "-Dbacktrace_vector_finish=__rbt_backtrace_vector_finish" "-Dbacktrace_vector_grow=__rbt_backtrace_vector_grow" "-Dbacktrace_vector_release=__rbt_backtrace_vector_release" "-Dbacktrace_close=__rbt_backtrace_close" "-Dbacktrace_open=__rbt_backtrace_open" "-Dbacktrace_print=__rbt_backtrace_print" "-Dbacktrace_simple=__rbt_backtrace_simple" "-Dbacktrace_qsort=__rbt_backtrace_qsort" "-Dbacktrace_create_state=__rbt_backtrace_create_state" "-Dbacktrace_uncompress_zdebug=__rbt_backtrace_uncompress_zdebug" "-o" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out/src/libbacktrace/alloc.o" "-c" "src/libbacktrace/alloc.c"
exit code: 0
TARGET = Some("x86_64-unknown-freebsd")
OPT_LEVEL = Some("0")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
CC_x86_64-unknown-freebsd = None
CC_x86_64_unknown_freebsd = None
HOST_CC = None
CC = None
HOST = Some("x86_64-unknown-freebsd")
TARGET = Some("x86_64-unknown-freebsd")
HOST = Some("x86_64-unknown-freebsd")
CFLAGS_x86_64-unknown-freebsd = None
CFLAGS_x86_64_unknown_freebsd = None
HOST_CFLAGS = None
CFLAGS = None
DEBUG = Some("true")
running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-I" "src/libbacktrace" "-I" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out" "-fvisibility=hidden" "-DBACKTRACE_ELF_SIZE=64" "-DBACKTRACE_SUPPORTED=1" "-DBACKTRACE_USES_MALLOC=1" "-DBACKTRACE_SUPPORTS_THREADS=0" "-DBACKTRACE_SUPPORTS_DATA=0" "-DHAVE_DL_ITERATE_PHDR=1" "-D_GNU_SOURCE=1" "-D_LARGE_FILES=1" "-Dbacktrace_full=__rbt_backtrace_full" "-Dbacktrace_dwarf_add=__rbt_backtrace_dwarf_add" "-Dbacktrace_initialize=__rbt_backtrace_initialize" "-Dbacktrace_pcinfo=__rbt_backtrace_pcinfo" "-Dbacktrace_syminfo=__rbt_backtrace_syminfo" "-Dbacktrace_get_view=__rbt_backtrace_get_view" "-Dbacktrace_release_view=__rbt_backtrace_release_view" "-Dbacktrace_alloc=__rbt_backtrace_alloc" "-Dbacktrace_free=__rbt_backtrace_free" "-Dbacktrace_vector_finish=__rbt_backtrace_vector_finish" "-Dbacktrace_vector_grow=__rbt_backtrace_vector_grow" "-Dbacktrace_vector_release=__rbt_backtrace_vector_release" "-Dbacktrace_close=__rbt_backtrace_close" "-Dbacktrace_open=__rbt_backtrace_open" "-Dbacktrace_print=__rbt_backtrace_print" "-Dbacktrace_simple=__rbt_backtrace_simple" "-Dbacktrace_qsort=__rbt_backtrace_qsort" "-Dbacktrace_create_state=__rbt_backtrace_create_state" "-Dbacktrace_uncompress_zdebug=__rbt_backtrace_uncompress_zdebug" "-o" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out/src/libbacktrace/backtrace.o" "-c" "src/libbacktrace/backtrace.c"
cargo:warning=src/libbacktrace/backtrace.c:37:10: fatal error: 'unwind.h' file not found
cargo:warning=#include "unwind.h"
cargo:warning=         ^~~~~~~~~~
cargo:warning=1 error generated.
exit code: 1

--- stderr
thread 'main' panicked at '

Internal error occurred: Command "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-I" "src/libbacktrace" "-I" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out" "-fvisibility=hidden" "-DBACKTRACE_ELF_SIZE=64" "-DBACKTRACE_SUPPORTED=1" "-DBACKTRACE_USES_MALLOC=1" "-DBACKTRACE_SUPPORTS_THREADS=0" "-DBACKTRACE_SUPPORTS_DATA=0" "-DHAVE_DL_ITERATE_PHDR=1" "-D_GNU_SOURCE=1" "-D_LARGE_FILES=1" "-Dbacktrace_full=__rbt_backtrace_full" "-Dbacktrace_dwarf_add=__rbt_backtrace_dwarf_add" "-Dbacktrace_initialize=__rbt_backtrace_initialize" "-Dbacktrace_pcinfo=__rbt_backtrace_pcinfo" "-Dbacktrace_syminfo=__rbt_backtrace_syminfo" "-Dbacktrace_get_view=__rbt_backtrace_get_view" "-Dbacktrace_release_view=__rbt_backtrace_release_view" "-Dbacktrace_alloc=__rbt_backtrace_alloc" "-Dbacktrace_free=__rbt_backtrace_free" "-Dbacktrace_vector_finish=__rbt_backtrace_vector_finish" "-Dbacktrace_vector_grow=__rbt_backtrace_vector_grow" "-Dbacktrace_vector_release=__rbt_backtrace_vector_release" "-Dbacktrace_close=__rbt_backtrace_close" "-Dbacktrace_open=__rbt_backtrace_open" "-Dbacktrace_print=__rbt_backtrace_print" "-Dbacktrace_simple=__rbt_backtrace_simple" "-Dbacktrace_qsort=__rbt_backtrace_qsort" "-Dbacktrace_create_state=__rbt_backtrace_create_state" "-Dbacktrace_uncompress_zdebug=__rbt_backtrace_uncompress_zdebug" "-o" "[redacted]/backtrace-rs/target/debug/build/backtrace-sys-528e7102cf580cf4/out/src/libbacktrace/backtrace.o" "-c" "src/libbacktrace/backtrace.c" with args "cc" did not execute successfully (status code exit code: 1).

', [redacted]/.cargo/registry/src/github.com-1ecc6299db9ec823/cc-1.0.15/src/lib.rs:2158:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

I've bisected this to 3449fe0e5b693332b4d2fd1806ab68c84520949f, which seems to have introduced this regression.

It seems like the configure script picked up that `unwind.h` wasn't available.

--- a/backtrace-sys/Cargo.toml
+++ b/backtrace-sys/Cargo.toml
@@ -16,3 +16,4 @@ libc = "0.2"
 
 [build-dependencies]
 cc = "1.0"
+pkg-config = "0.3"

--- a/backtrace-sys/build.rs
+++ b/backtrace-sys/build.rs
@@ -1,3 +1,4 @@
+extern crate pkg_config;
 extern crate cc;
 
 use std::env;
@@ -45,6 +46,12 @@ fn main() {
             build.define("BACKTRACE_ELF_SIZE", "64");
         } else {
             build.define("BACKTRACE_ELF_SIZE", "32");
+        }
+    }
+
+    if let Ok(unwind) = pkg_config::Config::new().probe("libunwind") {
+        for path in unwind.include_paths {
+            build.include(path);
         }
     }
 

